app/src/pages/index.tsx

```diff
@@ -15,6 +16,7 @@ export default function Home() {
                     <Stack gap={3}>
                         <PageHeader>CL311 Home Page</PageHeader>
                         <p>Welcome to CodeDay Labs 311!</p>
+                        <DeleteIcon/><DeleteIcon color="primary" /><DeleteIcon color="secondary" /><DeleteIcon color="success" /><DeleteIcon color="action" /><DeleteIcon color="disabled" />
```

Comment:
maybe put the code in separate lines for readability. other than that, it looks good!